### ISSUE TITLE
Join GNOME project

### ISSUE THREAD

**Issue by mrmcq2u** [openpaperwork/paperwork#627]
GNOME designers have recently started experimenting with document scanner mockups, might be good to reach out to them as open paperwork seems like it would fit the bill. 

Github.com/gnome-design-team/gnome-mockups/blob/52fb787932e7d2e3829243789b7f77e05c26dbc3/document-scanner/document-scanner.png

**Comment by jflesch** [comment]
I'll try again reaching them. ([I already did try in the past](https://mail.gnome.org/archives/gnome-list/2017-March/msg00009.html))

**Comment by jbicha** [comment]
[Simple Scan](https://git.gnome.org/browse/simple-scan/) is likely the GNOME scanner app.

**Comment by jflesch** [comment]
@jbicha you think those mockups are actually an attempt at reworking simple-scan ?

**Comment by jbicha** [comment]
@jflesch Yes, simple-scan was recently [added](https://git.gnome.org/browse/jhbuild/commit/?id=37ee388ed) as a GNOME Core App.

**Comment by jflesch** [comment]
Ok, good to know

**Comment by mrmcq2u** [comment]
That's weird, can't find any announcement of simple scan becoming a gnome
core app. Is it even part of gnome project?

**Comment by jbicha** [comment]
I believe there's no announcement because the change was made after the development snapshot 3.25.1 release. I expect it to be mentioned in the 3.25.2 development release and more prominently when 3.26 is released.

Yes, simple-scan is part of [GNOME](https://git.gnome.org/browse/simple-scan/). See the link in my earlier comment. Those mockups were made with Simple Scan [in mind](https://mail.gnome.org/archives/desktop-devel-list/2017-May/msg00005.html).

**Comment by jflesch** [comment]
https://mail.gnome.org/archives/desktop-devel-list/2017-May/thread.html#00004

**Comment by jflesch** [comment]
https://groups.google.com/forum/#!topic/paperwork-gui/Um91qIMd864

**Comment by jbicha** [comment]
You definitely do not have to shut down your website.

Here's some GNOME-related projects with websites:

- [GTK+](https://www.gtk.org/)
- [Glom](https://www.glom.org/)

**Comment by jbicha** [comment]
And [GCompris](http://gcompris.net) charges for their full Android and Windows versions. Although I guess they have switched to Qt now, it wasn't a problem when they were hosted on GNOME.

**Comment by jflesch** [comment]
Hm, interresting

**Comment by jflesch** [comment]
I'll have a discussion later with Gnome developers regarding the "do and don't" in Gnome apps. I think I'll simply adapt my plans accordingly.

**Comment by jflesch** [comment]
Here are the things I think must be done to become part of Gnome:
- [x] Flatpaking of Paperwork (#559)
- [ ] Appstream data for Paperwork (#690)
- [ ] Publication on Flathub (#691)
- [ ] Can be used in gnome-builder
- [ ] Hosted on gitlab.gnome.org
